utils/database.go

```diff
@@ -44,26 +26,33 @@ func InitialMigration() {
 
 // GetDB Get Database
 func GetDB() *gorm.DB {
-	DatabaseUsername := os.Getenv("DATABASE_USERNAME")
-	DatabasePassword := os.Getenv("DATABASE_PASSWORD")
-	DatabaseName := os.Getenv("DATABASE_NAME")
-	DatabaseHost := os.Getenv("DATABASE_HOST")
-	DatabasePort := os.Getenv("DATABASE_PORT")
-
-	newURI := "host=" + DatabaseHost + " port=" + DatabasePort + " user=" + DatabaseUsername + " dbname=" + DatabaseName + " sslmode=disable password=" + DatabasePassword
-	db, err := gorm.Open("postgres", newURI)
-	if err != nil {
-		log.Err(err).Msg("Database Error")
-		panic(err)
+	isDev := os.Getenv("DEV") == "true"
+
+	if !isDev {
+		DatabaseUsername := os.Getenv("DATABASE_USERNAME")
+		DatabasePassword := os.Getenv("DATABASE_PASSWORD")
+		DatabaseName := os.Getenv("DATABASE_NAME")
+		DatabaseHost := os.Getenv("DATABASE_HOST")
+		DatabasePort := os.Getenv("DATABASE_PORT")
+
+		newURI := "host=" + DatabaseHost + " port=" + DatabasePort + " user=" + DatabaseUsername + " dbname=" + DatabaseName + " sslmode=disable password=" + DatabasePassword
+		db, err := gorm.Open("postgres", newURI)
+
+		if err != nil {
+			log.Err(err).Msg("Database Error")
+			panic(err)
+		}
+
+		return db
+	} else {
```

Comment:
Since the if block returns a value, else shouldnt be a need

Reply:
Fixed in latest commit.